app/views/newsletters/index.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="newsletter-text">
+  <p>
+    Want to be kept up to date with the latest news from Any Rat Rescue? 
+  </p>
+  <button class="subscribe"><%= link_to "Subscribe", "#" %></button>
```

Comment:
So I have changed the line to be 
```erb
<%= link_to "Subscribe", "#", class: "button" %>
```
but that just gives me a link. It no longer looks like a button. Am I missing something?